File: stepview/src/main/java/me/jerryhanks/stepview/interfaces/TimeLineViewCallback.java
package me.jerryhanks.stepview.interfaces;

import android.widget.FrameLayout;

/**
 * @author <@Po10cio> on 10/18/17 for StepViewApp
 */

public interface TimeLineViewCallback<T> {
    void onBindView(T model, FrameLayout container);
}

File: timelineview/src/main/java/me/jerryhanks/timelineview/interfaces/TimeLineViewCallback.kt
package me.jerryhanks.timelineview.interfaces

import android.view.View
import android.widget.FrameLayout

import me.jerryhanks.timelineview.model.TimeLine

/**
 * @author <@Po10cio> on 10/18/17 for StepViewApp
 */

interface TimeLineViewCallback<in T : TimeLine> {
    fun onBindView(model: T, container: FrameLayout, position: Int): View
}
